packages/twenty-front/src/modules/serverless-functions/constants/IndexFilePath.ts

```diff
@@ -0,0 +1,3 @@
+import { join } from 'path';
```

Comment:
The path module comes from Node.js and shouldn't be used in the browser. I think it will work because Vite polyfills these modules, but it should be avoided.

If you want to manipulate URLs, you can use the [`URL` constructor](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL), or directly concatenate both parts as I think Windows support is not a top-priority.

See https://vite.dev/guide/troubleshooting.html#module-externalized-for-browser-compatibility.